.circleci/config.yml

```diff
@@ -97,7 +97,7 @@ jobs:
           command: npx cypress install
       - run:
           name: Run the node app.
-          command: npm run start-feature-no-webpack --env=cypress.env
+          command: npm run start-feature --env=cypress.env
```

Comment:
Need webpack for tests now, as we point to the react file from than just static.